tests/util.go

```diff
@@ -79,38 +80,32 @@ func RemoveNamespaces() {
 
 func ProcessTemplateWithParameters(srcFilePath, dstFilePath string, params ...string) string {
 	By(fmt.Sprintf("Overriding the template from %s to %s", srcFilePath, dstFilePath))
-	args := []string{"process", "-f", srcFilePath}
-	for _, v := range params {
-		args = append(args, paramFlag)
-		args = append(args, v)
-	}
-	out, err := ktests.RunOcCommand(args...)
-	Expect(err).ToNot(HaveOccurred())
+	out := execute(Result{verb: "process", filePath: srcFilePath, params: params})
```

Comment:
was that decision made regarding to `oc/kubectl` execution already?

Reply:
I don't know.